### PR TITLE
chore(graphql_formatter): performance tests

### PR DESCRIPTION
**Pull request by denbezrukov**
## Summary

Performance tests

### CODE REVIEW

**Comment by denbezrukov** [comment]
is it fine?O_o

**Comment by Sec-ant** [comment]
This seems a known bug, fyi: #3159

File: xtask/bench/benches/graphql_formatter.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#[global_allocator]
+static GLOBAL: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
+
+// Jemallocator does not work on aarch64 with musl, so we'll use the system allocator instead
+#[cfg(all(target_env = "musl", target_os = "linux", target_arch = "aarch64"))]
+#[global_allocator]
+static GLOBAL: std::alloc::System = std::alloc::System;
+fn bench_graphql_formatter(criterion: &mut Criterion) {
+    let mut all_suites = HashMap::new();
+    all_suites.insert("graphql", include_str!("libs-graphql.txt"));
```
Comment (ematipico) [inline xtask/bench/benches/graphql_formatter.rs:21]: Currently, we have an issue. Files are stored and saved using their name, but the graphql files have the same name, so only the first one of the list is benched

**Comment by ematipico** [comment]
That's fixed now. You should rebase your branch

**Review by ematipico — COMMENTED** [review]
You need to add these tests to the `Cargo.toml` file:

https://github.com/biomejs/biome/blob/a512d8104550ad0d2f0d29de9028a8e0b3d70689/xtask/bench/Cargo.toml#L78-L82

Then we can merge it

**Comment by denbezrukov** [comment]
@ematipico Thank you!

**Pull request merged**